lib/src/set.dart

```diff
@@ -169,14 +177,12 @@ class ManagedRealmSet<T extends Object?> with RealmEntity, SetMixin<T> implement
 
   @override
   bool contains(covariant T element) {
-    if (element is RealmObject) {
-      if (!element.isManaged) {
-        return false;
-      }
-
-      _ensureManagedByThis(element, "contains");
+    if (!_isManagedRealmObject(element)) {
+      return false;
     }
 
+    _ensureManagedByThis(element, "contains");
```

Comment:
"contains operation" will probably sounds better after the whole error message is built.

Reply:
I used what we use for lists as well. So lets keep it as is

Reply:
fixed the error text